numpy/core/_ufunc_config.py

```diff
@@ -3,13 +3,8 @@
 
 This provides helpers which wrap `umath.geterrobj` and `umath.seterrobj`
 """
-try:
-    # Accessing collections abstract classes from collections
-    # has been deprecated since Python 3.3
-    import collections.abc as collections_abc
-except ImportError:
-    import collections as collections_abc
 import contextlib
+from collections.abc import Callable
```

Comment:
I think I'd be inclined to use `import collections.abc` here, and use `collections.abc.Callable` below - its closer to the original intent of the code.

Reply:
I was considering this until I saw that there is only one item imported from `collections.abc` (either `Callable` or `Hashable`). Also, the modified lines are 16 characters shorter. However I can switch back to use `collections.abc.Callable` (etc.) if there is a strong desire to do so.

Reply:
Given that almost all the modules use this name just once, I think the fully-qualified name is more useful to a reader - it's also what we used to to prior to #10743. If you feel strongly then I'm happy to let another reviewer tie-break

Reply:
Ok, I'll agree that a fully-qualified name is more clear to some readers, so I've switched these back.